fair_mango/dataset/dataset.py

```diff
@@ -204,18 +432,60 @@ def get_real_target_for_all_groups(self) -> list[dict]:
             )
         return self.groups_real_target
 
-    def get_real_target_for_one_group(self, sensitive: Sequence[str]) -> pd.DataFrame:
-        """Retrieve real target for a specific group
+    def get_real_target_for_one_group(
+        self, sensitive: Sequence[str]
+    ) -> pd.Series | pd.DataFrame:
+        """Retrieve the real target corresponding to a specific demographic
+        group present in the sensitive features.
 
         Parameters
         ----------
         sensitive : Sequence[str]
-            the sensitive group
+            sequence of column names corresponding to sensitive features
+            (Ex: gender, race...).
 
         Returns
         -------
-        pd.DataFrame
-            the dataframe corresponding to the sensitive group
+        pd.Seies | pd.DataFrame
+            the pandas series or dataframe corresponding to the sensitive group
```

Comment:
question/suggestion : does "demographic" groups and "sensitive" groups mean the same thing ? If so you could keep only one for consistency

Reply:
They are the same and they are used interchangeably. A demographic group or a sensitive group is a subset of a population that is identified based on specific characteristics. They are both widely used in the field of fairness in AI and it is important for anyone working on the subject to know both of them.

Reply:
suggestion: if it refers to the same thing like you say, I think it is better to use only one in your description and variable names (my preference goes toward `sensitive` for the reasons introduced earlier 😁). 
I don't see a reason to interchange the term in the codebase. However, by using only one term, it will slightly ease the accessibility needed to understand this library. If I had this question, other users that might be less or more expert in AI/fairness might also have it.

Reply:
In my opinion it is better to use the two terms. They are not very advanced or hard to understand and I believe they can be understood from the docstrings alone. @MarcBresson do you mind giving us your opinion?

Reply:
@MarcBresson up

Reply:
oops, sorry.

I agree with @enzo-damion, I do prefer the use of sensitive. Fairness is already complex, and as Enzo pointed out, not only for people.

Reply:
Alright, I will update the documentation to use only `sensitive group`.